Dropdown/Dropdown.js

```diff
@@ -337,7 +337,7 @@ const DropdownBase = kind({
 			<div {...calcAriaProps} {...rest}>
 				{title}
 				<DropdownButton
-					aria-label={ariaLabel}
+					aria-label={ariaLabel || placeholder + ` ${$L('Dropdown')}`}
```

Comment:
You should read "Dropdown Button" when the user sets `aria-label`.

Reply:
It's fixed to read "Dropdown Button" even when the user sets `aria-label`.